doc/getting_started/installation_mac.rst

```diff
@@ -3,10 +3,6 @@
 Install macOS Host Dependencies
```

Comment:
This looks more like a troubleshooting section than "alternative instructions". It's duplication of the content in Getting Started plus some more information - maybe these sections should focus on troubleshooting?